darts/models/torch_forecasting_model.py

```diff
@@ -309,20 +337,41 @@ def fit(self,
             tb_writer.flush()
             tb_writer.close()
 
-    def predict(self, n: int) -> TimeSeries:
+    def predict(self, n: int, use_full_output_length: bool = False) -> TimeSeries:
```

Comment:
done

Reply:
Thanks it's better, however I think it can still be improved. "whether or not the full output sequence of the model prediction should be used to produce the output of this function" doesn't say too much.
If it's used, how is it used? And if it's not used, how is the output obtained?

Reply:
The more detailed description of the mechanism: "If `use_full_output_length` is set to `False`, the model will perform..." (line 335) is included in the block of text above the argument descriptions. Do you think I should move it to the description of `use_full_output_length`, or is it also not detailed enough?

Reply:
Ohhh sorry I missed this. That's all good then :)